test/e2e/exec_test.go

```diff
@@ -34,6 +28,15 @@ var _ = Describe("Podman exec", func() {
 		session := podmanTest.Podman([]string{"exec", "test1", "ls"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(ExitCleanly())
+
+		// With no command
+		session = podmanTest.Podman([]string{"exec", "test1"})
+		session.WaitWithDefaultTimeout()
+		expectedStatus := 255
```

Comment:
Why do we exit with 255? This is not documented, not for this PR but maybe worth filing an issue for?

Reply:
#22329